cdc/model/changefeed.go

```diff
@@ -45,6 +45,7 @@ type FeedState string
 // All FeedStates
 const (
 	StateNormal   FeedState = "normal"
+	StateError    FeedState = "error"
```

Comment:
By adding a new state, does it cause compatibility issues?

Reply:
no, it will not cause compatibility issues.